site/docs/guidelines/interaction-states.mdx

```diff
@@ -49,22 +49,23 @@ A disabled state applies to an element when the action is unavailable to a user.
   - If you need to disable a button, help users (including people using assistive technologies, such as screen reader) understand how to enable it again. For example, include nearby microcopy that explains why it is disabled and what action the user can take e.g. “There are no more reports to share. To share more reports, wait for a survey to close.”
 
 
-### Read-only
+### Read-only State
 
```

Comment:
Our capitalisation rules are a bit of a mess, but I think "State" should be lower case.